packages/xo-web/src/xo-app/settings/remotes/index.js

```diff
@@ -249,6 +250,30 @@ const COLUMNS_SMB_REMOTE = [
 
 const COLUMNS_S3_REMOTE = [
   COLUMN_NAME,
+  {
+    itemRenderer: remote => <Toggle disabled value={remote.protocol === 'http'} />,
```

Comment:
You don't want the user to be able to change that after the creation?

Reply:
no because it's very slow to give a feedback ( a few seconds waiting for the timeout) and toggling it a few time leads to unpredictable resuls